DataManagementSystem/DB/FileCatalogComponents/DatasetManager.py

```diff
@@ -19,7 +16,7 @@
 
 class DatasetManager:
```

Comment:
inherit from object? Also some documentation of what this class is used for may be helpful (I know you didn't make it, just since it is reviewed...)